mindsdb/integrations/handlers/bard_handler/bard_handler.py

```diff
@@ -0,0 +1,103 @@
+import os
+from typing import Dict, Optional
+
+import google.generativeai as genai
+import pandas as pd
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.utilities import log
+from mindsdb.utilities.config import Config
+
+logger = log.getLogger(__name__)
+
+
+class BardHandler(BaseMLEngine):
+    """
+    Integration with the Anthropic LLM Python Library
+    """
+
+    name = "bard"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default_chat_model = "gemini-pro"
+        self.supported_chat_models = ["gemini-pro"]
+        # self.default_max_tokens = 100
```

Comment:
If this is not needed, then please remove it.

Reply:
Yes sure 
but in future it might be needed ( there are some more models by google genai API but they are not yet available )